on some arches we use lib64 for libdir so hardcoding to lib
is not gonna work always

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: classpath-0.99/resource/Makefile.am
===================================================================
--- classpath-0.99.orig/resource/Makefile.am	2006-01-12 12:15:43.000000000 -0800
+++ classpath-0.99/resource/Makefile.am	2012-11-21 14:28:02.433005139 -0800
@@ -1,7 +1,7 @@
 ## used by automake to generate Makefile.in
 
 logging_DATA = java/util/logging/logging.properties
-loggingdir = $(prefix)/lib
+loggingdir = $(libdir)
 
 security_DATA = java/security/classpath.security
-securitydir = $(prefix)/lib/security
+securitydir = $(libdir)/security
